ext/RMagick/rmfill.c

```diff
@@ -267,7 +267,7 @@ horizontal_fill(
     // each of the "real" columns.
     master = ALLOC_N(PixelPacket, image->rows);
 
-    for (y = 0; y < image->rows; y++)
+    for (y = 0; y < (ssize_t) image->rows; y++)
```

Comment:
unnecessary type cast.
```suggestion
    for (y = 0; y < image->rows; y++)
```